server/src/main/java/com/objectcomputing/checkins/services/feedback_request/FeedbackRequest.java

```diff
@@ -0,0 +1,177 @@
+package com.objectcomputing.checkins.services.feedback_request;
+
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Nullable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "feedback_requests")
+public class FeedbackRequest {
+    @Id
+    @Column(name = "id")
+    @AutoPopulated
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "unique id of the feedback request", required = true)
+    private UUID id;
+
+    @Column(name = "creatorId")
+    @NotNull
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "id of the feedback request creator", required = true)
+    private UUID creatorId;
+
+    @Column(name = "requesteeId")
+    @NotNull
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "id of the person who is getting feedback requested on them", required = true)
+    private UUID requesteeId;
+
+    @Column(name = "templateId")
+    @NotNull
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "id of the template attached to request", required = true)
+    private UUID templateId;
+
+    @Column(name="sendDate")
```

Comment:
is this a nullable column ?

Reply:
No, it is not. I just added the `@NotNull` annotation to that column